localization/replace/english/wc_regiment_l_english.yml

```diff
@@ -14,3 +14,5 @@
  #Regional MaA Descriptions
  war_elephant_flavor:0 "#F A charging elekk is almost impossible to stop, making them excellent shock troops.#!"
  
+ druzhina_flavor:0 "#F Druzhina are skilled bodyguards who have devoted themselves to protecting their allies.#!"
```

Comment:
this is a vanilla MaA, do we really need this line?

Reply:
I think its fine